Abstracts/Repositories/Repository.php

```diff
@@ -18,6 +18,8 @@
 {
 
     use PrettusCacheableRepository;
+    
+    protected $maxPaginationLimit;
```

Comment:
shall we set this to `null` by default?

Reply:
Isn't it automatically null if you don't define it?

$maxPaginationLimit = null equals $maxPaginationLimit;

Reply:
yeah, but i think, that `isset($this->maxPaginationLimit)` and `empty($this->maxPaginationLimit)` behave different if the value is not set - but not sure on this one..

Or shall we just set the `maxPaginationLimit` by default to `0` to "disable" it.. 
And the `if-statement` down below should check, if `$this->maxPaginationLimit` is an integer and `>0`

what do you think?!

Reply:
added the changes you proposed with 0 as default value :) Will you add documentation please?